src/app/components/user/hidUsers.controller.js

```diff
@@ -0,0 +1,135 @@
+hidControllers.controller('hidUsersCtrl', ['$scope', '$rootScope', '$location', '$routeParams', 'gettextCatalog', 'alertService', 'hrinfoService', 'userService', 'User', 'List',
```

Comment:
I didn't add it to user.controllers.js as this is 615 lines long without it, which is far far too long already. 

It's the controller for the hidUsers directive so thought the name made sense.